### ISSUE TITLE
Unable to retrieve @root context in partial template

### ISSUE THREAD

**Issue by mgesmundo** [nolimits4web/Template7#11]
The registered partial template can't access the @root context:

```html
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Template7 bug</title>
</head>
    <body>
        <div id="template"></div>
        <script src="../dist/template7.min.js"></script>
        <script>
            T7bug = {
                context: {
                    rootvalue: 'this is missing',
                    items: [
                        {
                            content: 'one'
                        },
                        {
                            content: 'two'
                        },
                        {
                            content: 'three'
                        }
                    ]
                },
                template: [
                    '<ul>',
                    '   {{#each items}}',
                    '       {{> partial}}',
                    '   {{/each}}',
                    '</ul>'
                ].join(''),
                partial: [
                    '<li>',
                    '    <div>rootvalue: {{@root.rootvalue}}',
                    '       <div>{{content}}</div>',
                    '    </div>',
                    '</li>'
                ].join('')
            };

            Template7.registerPartial('partial', T7bug.partial);
            var template = Template7.compile(T7bug.template);
            var html = template(T7bug.context);
            document.getElementById('template').innerHTML = html;

        </script>
    </body>
</html>
```
In the example above the <code>@root.rootvalue</code> is not rendered.

**Comment by nolimits4web** [comment]
Fixed in 1.1.0

**Comment by mgesmundo** [comment]
Thanks Vladimir!
